Cython/Build/Tests/TestCyCache.py

```diff
@@ -65,17 +66,15 @@ def test_cycache_switch(self):
             msg='\n'.join(list(difflib.unified_diff(
                 a_contents.split('\n'), a_contents1.split('\n')))[:10]))
 
-    @unittest.skipIf(sys.version_info[:2] == (3, 12) and sys.platform == "win32",
-                     "This test is mysteriously broken on Windows on the CI only "
-                     "(https://github.com/cython/cython/issues/5825)")
     def test_cycache_uses_cache(self):
         a_pyx = os.path.join(self.src_dir, 'a.pyx')
         a_c = a_pyx[:-4] + '.c'
         with open(a_pyx, 'w') as f:
             f.write('pass')
         self.fresh_cythonize(a_pyx, cache=self.cache_dir)
         a_cache = os.path.join(self.cache_dir, os.listdir(self.cache_dir)[0])
-        gzip.GzipFile(a_cache, 'wb').write(b'fake stuff')
+        with closing(gzip.GzipFile(a_cache, 'wb')) as gzipfile:
+            gzipfile.write(b'fake stuff')
```

Comment:
Do we need `closing()` for this? AFAIR, context manager support was added to `GzipFile` somewhat early in the Py3 release series.

Reply:
I've pushed a quick update in https://github.com/cython/cython/commit/2f314fb1b1fd0a93eca08d842a7616e5c804105d.

It's in the docs now that I look. I'd failed to find it first time round.